test/button/button-subtle-perceptual.js

```diff
@@ -0,0 +1,73 @@
+const puppeteer = require('puppeteer');
```

Comment:
Is `puppeteer` needed here, or is it available on `ctx.puppeteer`?

Reply:
Yeah, currently.  The `ctx.puppeteer` is just an object that has a couple of Puppeteer specific helpers on it.  Initially I had all that helper logic in this file eventually factored it out in order to keep make the consumer's tests code simpler.  Aside from that, the `visual-diff` module doesn't have any hard dependency on Puppeteer.

Reply:
These are the helper functions: https://github.com/dbatiste/visual-diff/blob/master/visual-diff.js#L93